docs/install_guides/installing-openvino-overview.md

```diff
@@ -13,46 +13,57 @@
 
 @endsphinxdirective
 
-Intel® Distribution of OpenVINO™ toolkit is a comprehensive toolkit for developing applications and solutions based on deep learning tasks, such as: emulation of human vision, automatic speech recognition, natural language processing, recommendation systems, etc. It provides high-performance and rich deployment options, from edge to cloud. Some of its advantages are:
+Intel® Distribution of OpenVINO™ toolkit is a comprehensive toolkit for developing applications and solutions based on deep learning tasks, such as emulation of human vision, automatic speech recognition, natural language processing, recommendation systems, and more. It provides high-performance and rich deployment options, from edge to cloud. Some of its advantages are:
 
-* Enabling CNN-based deep learning inference on the edge.
-* Supporting various execution modes across Intel® technologies: Intel® CPU, Intel® Integrated Graphics, Intel® Neural Compute Stick 2, and Intel® Vision Accelerator Design with Intel® Movidius™ VPUs.
-* Speeding time-to-market via an easy-to-use library of computer vision functions and pre-optimized kernels.
+* Enables CNN-based and transformer-based deep learning inference on the edge or cloud.
+* Supports various execution modes across Intel® technologies: Intel® CPU, Intel® Integrated Graphics, Intel® Discrete Graphics, Intel® Neural Compute Stick 2, and Intel® Vision Accelerator Design with Intel® Movidius™ VPUs.
+* Speeds time-to-market via an easy-to-use library of computer vision functions and pre-optimized kernels.
```

Comment:
@ilya-lavrenov it is not clear to me what we mean with this. Do we need this sentence here?

Reply:
Hi @helena-intel , which sentence are you referring to? All three?